### ISSUE TITLE
 Avatar Group: Tooltip is missing for all the avatar

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#5047]
Description:Tooltip is missing for all the avatar

1. Steps:
2. Login to Fundamental ngx
3. Select the core
4. Select the Avatar group
5. Move the mouse cursor to the avatar
6. Toot tip is not shown 
7. As per the accessibility requirement all the UI icon or avatar should provide the tooltip. 

Expected: Tooltip should be provided for all the avatars

Attached the video 
[Screen Recording 2021-06-02 at 16.46.59.zip](https://github.com/SAP/fundamental-ngx/files/6584031/Screen.Recording.2021-06-02.at.16.46.59.zip)

**Comment by puru-hk** [comment]
Tested in this URL https://deploy-preview-5074--fundamental-ngx.netlify.app/#/core/avatar-group. it is working as expected. Hence closing this issue.
